pkg/agent/openflow/pipeline.go

```diff
@@ -461,7 +446,7 @@ func (c *client) defaultFlows() (flows []binding.Flow) {
 
 // tunnelClassifierFlow generates the flow to mark traffic comes from the tunnelOFPort.
 func (c *client) tunnelClassifierFlow(tunnelOFPort uint32, category cookie.Category) binding.Flow {
-	return c.pipeline[ClassifierTable].BuildFlow(priorityNormal).
+	return getTableByTableID(ClassifierTable).BuildFlow(priorityNormal).
```

Comment:
I wanted to remove this lookup when adding a flow.

What you think about these two ideas:

1. Add a new struct like pipelineTable, which saves ID and name, as well as binding.Table. It is defined with only ID and name initially, and we add a func createPipelineTable() which takes a pipelineTable argument, and creates a binding.Table (with ID and name) and saves it back to pipelineTable, and caches pipelineTable into flowTableCache/pipeline.
2. Similar to 1), but we extend binding.Table rather than add a new pipelineTable struct. So, we can new a binding.Table with only ID and name, but binding.CreateTable() or maybe binding.Table.Create() will really create the table in OVS.

In either case, line 44 - 72 will declare pipelineTable or binding.Table with ID and name, and all flow funcs can directly use the table to build flows, and need not to look up the table from ID.

Reply:
Sure, we could remove this lookup when adding flow entries. And I think the cache for all tables is still needed, because we need it to find a table for PacketIn case and antctl validation for the input table_name. What do you think?

Reply:
Yes, we still need the indexes with ID and name (for name, if it is not in the critical paths, we can also iterate all tables to look up by name).

Reply:
Got it, I will update.